ggshield/core/config/utils.py

```diff
@@ -55,7 +55,7 @@ def save_yaml(data: Dict[str, Any], path: str) -> None:
 
 
 def get_auth_config_dir() -> str:
-    return user_config_dir(appname="ggshield", appauthor="GitGuardian")
+    return get_config_dir()
```

Comment:
I think you can replace calls to `get_auth_config_dir()` with `get_config_dir()` and remove it.